pyproject.toml

```diff
@@ -45,18 +45,20 @@ repository = "https://github.com/pastas/pastastore"
 documentation = "https://pastastore.readthedocs.io/en/latest/"
 
 [project.optional-dependencies]
+full = ["pastastore[arcticdb,optional]"]
 lint = ["black", "flake8", "isort"]
 optional = ["contextily", "pyproj", "adjustText"]
 test = [
     "pastastore[arcticdb,lint,optional]",
-    "hydropandas",
+    "hydropandas[full]",
     "coverage",
     "codecov",
     "pytest",
     "pytest-cov",
     "pytest-dependency",
     "pytest-benchmark",
     "codacy-coverage",
+    "lxml",  # temporary fix: for hydropandas 0.8.0
```

Comment:
we can remove this if a new version of HydroPandas is released

Reply:
https://github.com/ArtesiaWater/hydropandas/commit/43f18a97e2fe1b9d30124b6155aaccaf37ee6839